autotest/utilities/test_gdal_translate.py

```diff
@@ -965,7 +965,7 @@ def test_gdal_translate_34(gdal_translate_path, tmp_path):
 # Test various errors (missing source or dest...)
 
 
-def test_gdal_translate_35(gdal_translate_path):
+def test_gdal_translate_35(gdal_translate_path, tmp_vsimem):
```

Comment:
I don't believe tmp_vsimem here and in the below tests of that file is useful, since it is used in gdaltest.runexternal_out_and_err() which is an external process. Thus plain /vsimem/ is fine. That said, that change probably doesn't hurt

Reply:
Agreed, the benefit here (and in many other changes) is just in a future `grep /vsimem/` or `grep tmp/` coming up clean.